app/views/categories/show.html.erb

```diff
@@ -0,0 +1,41 @@
+<div class="flex flex-col justify-center  h-full bg-white">
+  <div class="bg-[#3778c2] flex justify-between items-center text-white p-4 py-6">
+       <%= link_to categories_path do %>
+          <%= image_tag("https://img.icons8.com/ios/50/circled-chevron-left.png", alt: "exit", width: 20, height: 20) %>
+        <% end %>
+      <h3> Transections </h3>
+      <h3> <%= button_to category_path(@category), method: :delete, data:{confirm: "Are you sure you want to delete item?"}, class:" opacity-50 hover:opacity-100" do %>
+           <%= image_tag("https://img.icons8.com/ios-glyphs/30/000000/filled-trash.png", alt: "exit", width: 20, height: 20) %>
+          <% end %> </h3>
+  </div>
+    <div class="flex justify-between items-center  border-b border-slate-300 p-4 py-2">
+    <div class="flex items-center gap-px">
```

Comment:
Well done on your transactions page. Design guidelines are followed, and new transaction form follows project requirements!!! 👏🏾💯

![image](https://github.com/JakariaJishan/Budget-App/assets/38088818/fc5c5d11-0e7e-4379-b50f-9e6ea2fe2008)